test/compilable/traits.d

```diff
@@ -54,3 +54,8 @@ static assert(__traits(isModule, b));
 // This is supposed to work even though we haven't directly imported imports.pkgmodule.
 static assert(__traits(isModule, imports.pkgmodule));
 static assert(!__traits(isModule, MyStruct));
+
+version(Windows)
```

Comment:
That's very few tests, although if you use the helper mentioned in the previous comment there should not be bad surprises.

Reply:
https://github.com/dlang/dmd/pull/10013#issuecomment-500272593
https://github.com/dlang/dmd/pull/10013#issuecomment-500272986

Reply:
About overloads: maybe they can be detected and in this case an error can be reported:
_"use __traits(getOverloads) with an index to specify a single symbol"_ or something like that.